site/app/controllers/UserProfileController.php

```diff
@@ -166,4 +167,36 @@ public function changeProfilePhoto() {
             }
         }
     }
+
+    /**
+     * @Route("/user_profile/change_secondary_email", methods={"POST"})
+     * @return JsonResponse
+     * @throws \ImagickException
```

Comment:
why would this throw a `\ImagickException` ?

Reply:
That's not supposed to be there. Will remove.